scripts/migrate_entity_nodes.py

```
@@ -0,0 +1,186 @@
+logging.basicConfig(level=logging.INFO, format="%(asctime)s %(levelname)s %(message)s")
+logger = logging.getLogger(__name__)
+
+ENTITY_TAG_RE = re.compile(r"^entity:([a-z]+):(.+)$")
+
+
+def slug_to_name(slug: str) -> str:
+    """Convert a slug like 'alice-smith' to 'Alice Smith'."""
+    return slug.replace("-", " ").title()
+
+
+def connect_falkordb() -> Any:
```

Ruff (0.15.5)

[error] Undefined name `Any`

(F821)